tests/test_envs.py

```diff
@@ -236,6 +237,38 @@ def sleep(duration: float) -> None:
         assert o.tool_call_id == t.id
 
 
+class SlowEnv(Environment[None]):
```

Comment:
Not a priority, but I think it'd be nice to switch all these one-off testing envs to use `fenv`. In this case I couldn't because `fenv` doesn't expose a way to pass arguments to `exec_tool_calls`, but that should be an easy change.